sdk/guestconfiguration/Azure.ResourceManager.GuestConfiguration/tests/Scenario/GuestConfigurationAssignmentCollectionTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.GuestConfiguration.Models;
+using Azure.ResourceManager.GuestConfiguration.Tests.Utilities;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.GuestConfiguration.Tests.Scenario
+{
+    [TestFixture]
```

Comment:
Please remove this attribute and re-record all the tests. We need to make sure that both of the sync and async methods are tested.

Reply:
removed